config/puma.rb

```
@@ -0,0 +1,202 @@
+#!/usr/bin/env puma
+
+port Integer(ENV.fetch('PORT'))
```

Any reason for not using `ENV['PORT']`?

---

To make it fail early if no port was specified? E.g. if someone forgets to run under `foreman`.